libs/langgraph/langgraph/prebuilt/__init__.py

```diff
@@ -12,6 +13,7 @@
 
 __all__ = [
     "create_react_agent",
+    "create_chain_executor",
```

Comment:
`create_chain_executor` or `create_chain` to mirror `create_react_agent` ?

Reply:
I don't have a strong opinion here, but less typing is usually better if there's no ambiguity